### ISSUE TITLE
Nicehash 5th July New Platform Command Lines

### ISSUE THREAD

**Issue by thebeerbaron2018** [RainbowMiner/RainbowMiner#592]
Today Nicehash has started transitioning to their new platform.  Currently you can mine on the old platform and will still be paid as the balance will carry forward.  As of tomorrow mining on the new platform will become live so you'll need to update your Nicehash mining address in the pools.config.txt file.  They have updated their API so this might need updating here too - and the same with the command line if they change their stratum addresses.  We won't know this until tomorrow but need to keep an eye on it

**Comment by thebeerbaron2018** [comment]
Live updates on their webpage at : https://www.nicehash.com/news/important-transition-to-a-new-nicehash-platform

**Comment by thebeerbaron2018** [comment]
... and live updates via twitter at : https://twitter.com/NiceHashMining

**Comment by thebeerbaron2018** [comment]
New API docs at : https://docs-test.nicehash.com/

**Comment by RainbowMiner** [comment]
Thank you - I am following the development closely.

**Comment by RainbowMiner** [comment]
We have now the following situation. Since the old platform is still more profitable, than the new one, I decided to go two-way:
- NiceHash pool got a selection "Platform" to select on which platform (v1 or v2) the used wallet address is registered. It currently defaults to v1
- I have added a temporary second pool "NiceHashV2" which exclusively mines on the new Nicehash platform.

That gives us the chance to mine on both platforms side-by-side and RainbowMiner decides the more profitable.
